FinSearchUnified/Resources/services.xml

```diff
@@ -12,16 +12,23 @@
     <services>
         <service id="fin_search_unified.subscriber.frontend" class="FinSearchUnified\Subscriber\Frontend">
             <argument>%fin_search_unified.plugin_dir%</argument>
-            <argument type="service" id="template" />
-            <tag name="shopware.event_subscriber" />
+            <argument type="service" id="template"/>
```

Comment:
Nope. Revert these changes. Don't touch anything that's not related to the story.